episodes/introduction.md

```diff
@@ -1,114 +1,66 @@
 ---
-title: "Using Markdown"
+title: "Task Management for Humans"
 teaching: 10
 exercises: 2
 ---
 
 :::::::::::::::::::::::::::::::::::::: questions 
 
-- How do you write a lesson using Markdown and `{sandpaper}`?
+- How can you use tools and frameworks to work more effectively, not so you can do more work for your boss but as a form of self care?
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 ::::::::::::::::::::::::::::::::::::: objectives
 
-- Explain how to use markdown with The Carpentries Workbench
-- Demonstrate how to include pieces of code, figures, and nested challenge blocks
+- Introduce various tools and frameworks.
+- Give examples of how these tools can be used to work more effectively.
+- Share case studies of how people have used these frameworks to improve their own mental health.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 ## Introduction
 
-This is a lesson created via The Carpentries Workbench. It is written in
-[Pandoc-flavored Markdown](https://pandoc.org/MANUAL.txt) for static files and
-[R Markdown][r-markdown] for dynamic files that can render code into output. 
-Please refer to the [Introduction to The Carpentries 
-Workbench](https://carpentries.github.io/sandpaper-docs/) for full documentation.
-
-What you need to know is that there are three sections required for a valid
-Carpentries lesson:
-
- 1. `questions` are displayed at the beginning of the episode to prime the
-    learner for the content.
- 2. `objectives` are the learning objectives for an episode displayed with
-    the questions.
- 3. `keypoints` are displayed at the end of the episode to reinforce the
-    objectives.
-
-:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
-
-Inline instructor notes can help inform instructors of timing challenges
-associated with the lessons. They appear in the "Instructor View"
+Developing software for research can be challenging. Many people work on multiple projects, often to tight deadlines, regularly working in fields that we know nothing about. On top of that, we have to do training, keep up with constantly evolving technologies and try to fit in extra activities to advance our careers. It will come as no surprise to most RSEs to hear that anxiety, burnout and impostor syndrome are common in this industry.
 
-::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+But what can we practically do to protect our mental health while working in this field? We have little control over our actual workloads, and telling a almost-burned out engineer to do some yoga or mindfulness exercises is seldom well received.
```

Comment:
please check - do we need a hyphen in "almost-burned out"?

Reply:
I'm not sure. It's probably either all (almost-burned-out) or nothing (almost burned out)...